src/ripple/app/paths/impl/DirectStep.cpp

```diff
@@ -635,8 +631,7 @@ DirectStepI<TDerived>::fwdImp (
     std::tie (maxSrcToDst, srcDebtDir) =
```

Comment:
Any reason not to use structured bindings here?

Reply:
No reason, I just missed converting this one. The major reason for some of the ones I didn't convert is you can't capture a structured binding in a lambda. But that doesn't apply here. Will fix.

Reply:
Yeah, it's unfortunate that we still can't capture structured bindings. It may get fixed in C++23 as there has been a [CWG issue](https://wg21.cmeerw.net/cwg/issue2308) for a while and there's no technical reason to disallow it AFAIK. [P1091](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1091r3.html) will fix this if it makes it into C++23.

Reply:
Fixed in 03289817b `[fold] Use structured binding in another place & rm std::pair ctor:`